analyzers/Merger.py

```diff
@@ -0,0 +1,19 @@
+from heppy.framework.analyzer import Analyzer
+import copy
+
+
+
+class Merger(Analyzer):
```

Comment:
Yes, sorry, meant docstring not doctest. 
doctest is a special kind of docstring, but we are not using that in this project.